### PR TITLE
Some minor changes to gatewayv2 documentation

### PR DESCRIPTION
**Pull request by wvandervaart**
Just some small fixes based of our PM's comments on the documentation.

### CODE REVIEW

File: docs/hackers/warp-client-config.md [hunk 1]
```diff
@@ -18,9 +18,11 @@ These guides assume you already have the WARP Client installed, if you haven't p
 1. Go to account preferences in the client.
 2. Fill in the team domain:
 ![image](./images/warp_win_teams.png)
+> **Note:** If you do not know the team domain please contact support.
 
-3. Login to the WARP authentication website (pops-up automatically)
+3. Login to the WARP authentication website (pops-up automatically). use your whitelisted e-mail address.
```
Comment (martijnrusschen) [inline docs/hackers/warp-client-config.md:23]:
```suggestion
3. Login to the WARP authentication website (pops-up automatically). Use your whitelisted e-mail address.
```

File: docs/hackers/warp-client-config.md [hunk 2]
```diff
@@ -37,9 +39,11 @@ Click "account" tab
 
 2. Fill in the team domain:
 ![image](./images/warp_team_name.png)
+> **Note:** If you do not know the team domain please contact support.
 
-3. Login to the WARP authentication website (pops-up automatically)
+3. Login to the WARP authentication website (pops-up automatically). use your whitelisted e-mail address.
```
Comment (martijnrusschen) [inline docs/hackers/warp-client-config.md:44]:
```suggestion
3. Login to the WARP authentication website (pops-up automatically). Use your whitelisted e-mail address.
```

File: docs/hackers/warp-client-config.md (hunk 1 above)
Comment (wvandervaart) [reply docs/hackers/warp-client-config.md]: Changed in latest commit

**Review by martijnrusschen — APPROVED** [review]

File: docs/hackers/warp-client-config.md (hunk 2 above)
Comment (wvandervaart) [reply docs/hackers/warp-client-config.md]: Changed in latest push

**Pull request merged**